backend/stakepoold/rpc/stakepoolrpc/api.pb.go

```diff
@@ -1913,10 +1913,10 @@ type StakepooldServiceClient interface {
 }
 
 type stakepooldServiceClient struct {
-	cc *grpc.ClientConn
+	cc grpc.ClientConnInterface
```

Comment:
omg, this is awesome. No we can test this. I wanted to manually change this. #feelsgoodman that google agrees with me.